apiary.apib

```diff
@@ -71,7 +73,8 @@ API endpoints that Operation Code's Rails backend makes available to its React f
                       "full_time": "false",
                       "hardware_included": "false",
                       "has_online": "true",
-                      "online_only": "false"
+                      "online_only": "false",
+                      "is_partner": false,
```

Comment:
Can you pls change all of these `true/false` in this endpoint from a string to a boolean (like yours).